apollo-router/src/plugins/demand_control/cost_calculator/fixtures/basic_supergraph_schema.graphql

```diff
@@ -0,0 +1,102 @@
+schema
```

Comment:
This is just the existing `basic_schema.graphql` run through rover compose to get a supergraph with a single subgraph.